packages/components/src/FormField/FormField.tsx

```diff
@@ -119,6 +122,7 @@ function FormFieldInternal(props: FormFieldInternalProps) {
     autoFocus: autofocus,
     onKeyDown: handleKeyDown,
   };
+  const inputRefs = mergeRefs([inputRef, fieldRef]);
```

Comment:
Uses the field ref from `useController` like we do in components-native [here](https://github.com/GetJobber/atlantis/blob/master/packages/components-native/src/InputText/InputText.tsx#L435-L438)